EDI/UserInterface/01a_Exports.md

```diff
@@ -0,0 +1,340 @@
+
+- *Source entity*   
+   Entity that defines the source you want to map to the destination fields of the EDIFACT message, for example the *Retail suite business document* entity. 
+
+- *Source attribute set*   
+   Attribute set that represents the source entity you want to map to the destination fields of the EDIFACT message, for example the *Retail suite business document* attribute set. 
+
+- *Sorting*   
+   Number that represents the sequence in which the source entities are processed.
+
+- ![Sort](../../Assets/Icons/Sort01.png "[Sort]") (Sort)   
+   Click this button and drag and drop the source entity to the position you prefer.
+
+- [CANCEL]    
+   Click this button to finish the sorting without saving it. The *Data sources (Sorting mode)* view is closed.
+
+- [SAVE]   
+   Click this button to save the sorting. The *Data sources (Sorting mode)* view is closed.
+
+## Data sources &ndash; Wizard for creating source entities and doing the mappings  
+
+The wizard window for creating source entities and doing the mappings is displayed when you add a new data source.
+
+### Step 1: Select source
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources > Click the Add button*
+
+![Select source](../../Assets/Screenshots/EDI/UserInterface/SelectSourceEntitySelection.png "[Data sources (Sorting mode)]")
+
+- *Entity type*    
+  Click the drop-down list and select the entity type you want to use for a later data mapping with the EDIFACT fields and segments.
+
+- *Attribute set*    
+   Attribute set that is assigned to the entity type. The attribute set drop-down list is automatically filled after you have selected an *Entity type*.
+
+- [CANCEL]    
+   Click this button to finish the wizard without saving it.    
+   The *Select source* wizard window is closed.
+
+- [CONTINUE]   
+   Click this button to go one step forward.    
+   The *Map attributes* wizard window is displayed (see below). 
+
+### Step 2: Map attributes
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources > Click an exporter definition*
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources > Click the Add button > Click the CONTINUE button*
+
+![Map attributes](../../Assets/Screenshots/EDI/UserInterface/MapAttributes.png "[Map attributes)]")
+
+This step is displayed when you assign a data source. It is also displayed when you select a data source for doing the mapping. For detailed information on the mapping functions, see [Mapping](../../DataHub/UserInterface/02d_ETL.md#mapping) in the *DataHub* documentation.
+
+>[Info] Note that the wizard does not provide the whole functional scope that the *DataHub* mapping provides. The wizard contains only those functions that are relevant to export message mappings.
+
+- [CANCEL]    
+   Click this button to finish the wizard without saving it.    
+   The wizard window is closed.
```

Comment:
The *Map attributes* wizard window is closed.

Reply:
Es geht darum, dass Cancel allgemein den Wirzard abbricht...